ipaserver/install/dsinstance.py

```diff
@@ -652,21 +652,23 @@ def __add_default_schemas(self):
             # Does not apply with newer DS releases
             pass
 
-    def start(self, *args, **kwargs):
-        super(DsInstance, self).start(*args, **kwargs)
+    def start(self, instance_name="", capture_output=True, wait=True):
+        super().start(instance_name, capture_output=capture_output, wait=wait)
```

Comment:
I see `ipa-4-6` label, this assumes Python2. `super()` is Python3 feature [0]

[0]: https://docs.python.org/3/whatsnew/3.0.html#builtins

Reply:
good call, fixed!